style.css

```
@@ -278,8 +281,34 @@ section > h2 {
   font-weight: normal;
 }
 
+.overview-flex-box {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 0 1.2em;
+}
+
+@media all and (min-width: 640px) {
+  .overview-flex-box {
+    display: flex;
+    flex-direction: row;
```

CSS Gridを使ってやると、子要素でのハードコードを削減 & レイアウト情報を親要素に集約できるのでお勧めです。
`.map` に対する `max-width: 600px;` の指定を削除できる。
また、 `.overview` と  `.map` がくっついてしまっている部分は `column-gap` で簡単に解決できますね。

```suggestion
    display: grid;
    grid-template-columns: 1fr 1fr;
    column-gap: 40px:
```